package.json

```diff
@@ -118,6 +118,7 @@
     "karma-chrome-launcher": "2.0.0",
     "karma-cli": "1.0.1",
     "karma-coverage": "1.1.1",
+    "karma-remap-coverage": "^0.1.4",
```

Comment:
please remove the ^ so that it does not autoupdate, we are maintaining a strict update policy